.pipelines/.vsts-vhd-builder-release-windows.yaml

```diff
@@ -8,7 +8,9 @@ pr:
   paths:
     include:
     - .pipelines/.vsts-vhd-builder-release-windows.yaml
+    - .pipelines/templates/.build-and-test-windows-vhd-template.yaml
```

Comment:
1. Do the tests pass now? It may block all PRs if the answer is no
2. Should the PR be created against dev branch by following the current process?

Reply:
1. This PR will not make the tests pass. Which I'm comfortable with. At least bypassing failing tests will be a conscious decision rather than an accident.
2. dev branch is now gone! So we can make PRs against master again.